templates/addons/params.yml

```diff
@@ -0,0 +1,9 @@
+App:
+  Type: String
+  Description: Your application's name.
+Env:
+  Type: String
+  Description: The environment name your service is being deployed to.
+Service:
```

Comment:
Ok, one more thing. In the hopes of growing in the future beyond just services, how about we call this `Name` instead. And it's the name of your service, job, workflow, whatever.